node/src/actor_system_factory.rs

```diff
@@ -1172,27 +1183,27 @@ mod tests {
         let system = System::new("MASQNode");
         System::current().stop();
         system.run();
-        check_bind_message(&recordings.dispatcher, false);
-        check_bind_message(&recordings.hopper, false);
-        check_bind_message(&recordings.proxy_client, false);
-        check_bind_message(&recordings.proxy_server, false);
-        check_bind_message(&recordings.neighborhood, false);
-        check_bind_message(&recordings.ui_gateway, false);
-        check_bind_message(&recordings.accountant, false);
-        check_pool_bind_message(&recordings.stream_handler_pool, 0); // what _should_ we be doing here?
-        check_pool_bind_message(&recordings.dispatcher, 1);
-        check_pool_bind_message(&recordings.neighborhood, 1);
-        check_new_ip_message(
+        check_for_initial_bind_message(&recordings.dispatcher, false);
+        check_for_initial_bind_message(&recordings.hopper, false);
+        check_for_initial_bind_message(&recordings.proxy_client, false);
+        check_for_initial_bind_message(&recordings.proxy_server, false);
+        check_for_initial_bind_message(&recordings.neighborhood, false);
+        check_for_initial_bind_message(&recordings.ui_gateway, false);
+        check_for_initial_bind_message(&recordings.accountant, false);
+        check_for_pool_bind_message_at(&recordings.stream_handler_pool, 0); // what _should_ we be doing here?
```

Comment:
Do you happen to remember if the comment at the end of the line is from you? Could you write it while you were working on this card? It sounds possible to me. Please decide the destiny of it yourself, but I think it won't do anybody any good in the future if they are missing the background story you have.

Reply:
I think it's obsolete--overtaken by events. I got rid of it.